src/components/media/MediaMenu.tsx

```diff
@@ -10,7 +11,15 @@ import { Menu } from '../menu';
 import { bdlGray50 } from '../../styles/variables';
 import './Media.scss';
 
-export interface MediaMenuProps extends PlainButtonProps {
+const messages = defineMessages({
+    commentOptionsArialLabel: {
+        id: 'be.contentSidebar.activityFeed.comment.commentOptionsArialLabel',
```

Comment:
Can we update the `id` property of this message should reflect its new host component, since it's not directly related to the activity feed now?

Reply:
```suggestion
        id: 'boxui.media.menuButtonArialLabel',
        ```

Reply:
thank you @alexkrolick